src/cli/Peers.ts

```diff
@@ -0,0 +1,387 @@
+import fs from 'fs';
+import commander from 'commander';
+import { PolykeyAgent, PeerInfo } from '../lib/Polykey';
+import {} from '../../tmp/hole-punch-test/publicserver';
+import { actionRunner, pkLogger, PKMessageType, determineNodePath } from '.';
+
+function makeAddPeerCommand() {
+  return new commander.Command('add')
+    .description('add a new peer to the store')
+    .option('-b, --base64 <base64>', 'decode the peer info from a base64 string')
+    .option('-p, --public-key <publicKey>', 'path to the file which contains the public key')
```

Comment:
Are we standardizing what kind of keys we are doing? This is one of the problems that wireguard solved by standardizing on ed25519 keys. We could do the same, but that means pk protocols must communicate the version of pk. In case we then upgrade versions. I like the wireguard way, protocol negotiation is a PITA.

Reply:
Yeah we are currently using the keybase default which is RSA, but they also have support for elliptic curve PGP keys. Actually I think kbpgp abstracts all that away, when you're boxing and unboxing you don't need to specify what kind of key is being used. We also we get versioning for free with kbpgp, the version is in each pgp message at the top.